crates/k8s-version/Cargo.toml

```diff
@@ -0,0 +1,23 @@
+[package]
+name = "k8s-version"
+version = "0.1.0"
+authors.workspace = true
+license.workspace = true
+edition.workspace = true
+repository.workspace = true
+
+[features]
+darling = ["dep:darling"]
```

Comment:
I added the feature assuming we eventually will publish this crate on crates.io. It is optional, because some people might not require that `Version` are parseable from attributes via darling. This is a rather use-case specific feature.

`stackable-versioned` makes use of this feature to enable direct parsing of `Version`s from attributes.